sdk/maps/Azure.Maps.Rendering/src/Models/RegionalCopyright.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Maps.Rendering
+{
+    /// <summary> The RegionalCopyright. </summary>
+    [CodeGenModel("RegionCopyrights")]
+    public partial class RegionalCopyright
+    {
+        /// <summary> Initializes a new instance of RegionalCopyright. </summary>
+        internal RegionalCopyright()
+        {
+            Copyrights = new ChangeTrackingList<string>();
+        }
+
+        /// <summary> Initializes a new instance of RegionalCopyright. </summary>
+        /// <param name="copyrights"> Copyrights array. </param>
```

Comment:
nit:  The comments for the parameters lack enough context to be helpful.   The use of `array` here is also inaccurate given the type.